src/Microsoft.Data.SqlClient/tests/ManualTests/Microsoft.Data.SqlClient.ManualTesting.Tests.csproj

```diff
@@ -184,6 +184,7 @@
     <Compile Include="SQL\Utf8SupportTest\Utf8SupportTest.cs" />
     <Compile Include="SQL\WeakRefTest\WeakRefTest.cs" />
     <Compile Include="SQL\WeakRefTestYukonSpecific\WeakRefTestYukonSpecific.cs" />
+    <Compile Include="TracingTests\TestTdsServer.cs" />
```

Comment:
If this is the functional test, why it is here?

Reply:
The entire `DiagnosticTest `was in functional tests. We moved it to Manual tests along with its dependencies and one of them is `TestTdsServer`. `TestTdsServer` are used in both places using in different tests. In other word, this has been used in both places in different tests. In manual tests it is for `DiagnosticTests` and in functional test it has been used for `SqlConnectionBasicTests`. The only Reason we had to do it, was connection string setup from pipelines and json file.

Reply:
Have a closer look in the repository, I see you are calling this TdsServer from DiagnosticTest. Maybe we can reorganize this a bit in the future to avoid duplicates between Functional and Manual tests.

Reply:
That certainly is doable. I had the option of adding dataTestUtility in Functional tests or add this single class to Manual Tests. for future references, we probably, just an idea, can add a common folder for tests and use them from there maybe?

Reply:
Yeah, I agree with the common folder to hold stuff for both functional and manual tests to avoid duplicates.